phpunit-integration-legacy-solr.xml

```diff
@@ -58,6 +58,7 @@
             <file>eZ/Publish/API/Repository/Tests/SearchServiceTranslationLanguageFallbackTest.php</file>
             <file>eZ/Publish/API/Repository/Tests/SearchServiceLocationTest.php</file>
             <file>eZ/Publish/API/Repository/Tests/SearchServiceAuthorizationTest.php</file>
+            <file>eZ/Publish/API/Repository/Tests/SearchServiceAggregationTest.php</file>
```

Comment:
There's the `eZ/Publish/API/Repository/Tests/SearchService` directory which can contain Aggregations sub-directory with smaller test cases.

Reply:
As agreed with @kaff: https://github.com/ezsystems/ezplatform-kernel/pull/94#discussion_r500115052